test/integration/integration_admin_test.cc

```diff
@@ -123,6 +123,11 @@ std::string ContentType(const BufferingStreamDecoderPtr& response) {
 } // namespace
 
 TEST_P(IntegrationAdminTest, Admin) {
+  config_helper_.addConfigModifier([&](envoy::config::bootstrap::v2::Bootstrap& bootstrap) -> void {
```

Comment:
Does this add anything to this test? We aren't verifying any behavior below AFAICT? This is why I have been asking if there is any way that we can actually verify that we are draining a listener. Even if there is no stat could we potentially look for an info log either in this test or in the proper integration test?

Reply:
Yes it does. It modifies the existing listener and sets the traffic direction as OUTBOUND. I have added the stat now - Now it should be clear. PTAL

Reply:
I also brought back the second test, because with stat we can validate that behaviour as well.